formcentral/Dockerfile.base

```diff
@@ -0,0 +1,15 @@
+FROM golang:alpine
+
+RUN apk add --no-cache \
+    gcc \
+    libc-dev
+
+RUN mkdir /app
+WORKDIR /app
+
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+COPY . /app
+RUN go build -o main .
```

Comment:
In a dev environment wouldn't you mount a volume with the local files so that you can edit them locally? If so, this shouldn't be necessary...